gulp/tasks/lint.js

```diff
@@ -5,6 +5,7 @@ var gulp = require( 'gulp' );
 var gulpEslint = require( 'gulp-eslint' );
 var handleErrors = require( '../utils/handle-errors' );
 var minimist = require( 'minimist' );
+var through = require( 'through2' );
```

Comment:
You might name the variable the same as the package so a search for the package turns up where it is used, though in this case it's so close it doesn't really matter. Also, all client-side scripts can move to es6 `const`, if you want.